app/views/smart_answers/visualise.html.erb

```diff
@@ -14,10 +14,15 @@
   var adjacencyList = <%= JSON.pretty_generate(@graph_data).html_safe %>;
 <%- end %>
 
-<%= render "govuk_publishing_components/components/title", {
-  context: "Flow visualisation for",
-  title: @title,
-} %>
+<div class="govuk-!-margin-top-8">
```

Comment:
I can see why this is necessary, but I'm not a fan of adding an extra wrapper to adjust the vertical spacing. Is there another way this could be done more cleanly? Looking at the surrounding markup it seems not. What does this page look like if there's no top margin added? Will it be inconsistent with the others?